docs/static/settings-file.asciidoc

```diff
@@ -282,6 +282,16 @@ Current options are:
   This setting is ignored unless `api.ssl.enabled` is set to `true`.
 | _N/A_
 
+| `api.ssl.supported_protocols`
+a| List of allowed SSL/TLS versions to use when establishing a secure connection. The available protocols depends on the JVM version, a few protocols are disabled by default and needs to be enabled manually by changing `jdk.tls.disabledAlgorithms` in the *$JDK_HOME/conf/security/java.security* configuration file.
```

Comment:
```suggestion
| List of allowed SSL/TLS versions to use when establishing a secure connection. The availability of protocols depends on the JVM version. Certain protocols are disabled by default and need to be enabled manually by changing `jdk.tls.disabledAlgorithms` in the *$JDK_HOME/conf/security/java.security* configuration file.
```

Reply:
I've applied the re-words suggestion and kept the removed `a` operator, it's necessary here to support lists (possible values) and blocks (styles) elements.